src/controller/datacontroller.js

```diff
@@ -247,13 +272,18 @@ export default class DataController {
 	/**
 	 * See {@link module:engine/controller/insertcontent~insertContent}.
 	 *
+	 * Note that data inserted by a data pipeline might contain temporary {@link module:engine/model/element/~Element elements}
+	 * marking {@link module:engine/model/document~Document#markers markers} ranges. We need to remove them because
+	 * data pipeline allows to set markers only by a {@link #set set method}.
+	 *
 	 * @fires insertContent
 	 * @param {module:engine/model/documentfragment~DocumentFragment} content The content to insert.
 	 * @param {module:engine/model/selection~Selection} selection Selection into which the content should be inserted.
 	 * @param {module:engine/model/batch~Batch} [batch] Batch to which deltas will be added. If not specified, then
 	 * changes will be added to a new batch.
 	 */
 	insertContent( content, selection, batch ) {
+		extractMarkersFromModelFragment( content );
```

Comment:
Why is it here? Any clean up should be done beforehand, using the clipboard events (`#paste` or `#clipboardInput`).